recipes-core/aziot-edged/aziot-edged_1.5.5.bb

```diff
@@ -0,0 +1,597 @@
+SRCREV_aziot-key-client = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-key-client"
+SRCREV_FORMAT .= "_aziot-key-client-async"
+SRCREV_aziot-key-client-async = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-key-client-async"
+SRCREV_FORMAT .= "_aziot-key-common"
+SRCREV_aziot-key-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-key-common"
+SRCREV_FORMAT .= "_aziot-key-common-http"
+SRCREV_aziot-key-common-http = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-key-common-http"
+SRCREV_FORMAT .= "_aziot-key-openssl-engine"
+SRCREV_aziot-key-openssl-engine = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-key-openssl-engine"
+SRCREV_FORMAT .= "_aziot-keyd-config"
+SRCREV_aziot-keyd-config = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-keyd-config"
+SRCREV_FORMAT .= "_aziot-keys-common"
+SRCREV_aziot-keys-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/key/aziot-keys-common"
+SRCREV_FORMAT .= "_aziot-tpmd-config"
+SRCREV_aziot-tpmd-config = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/tpm/aziot-tpmd-config"
+SRCREV_FORMAT .= "_aziotctl-common"
+SRCREV_aziotctl-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/aziotctl/aziotctl-common"
+SRCREV_FORMAT .= "_cert-renewal"
+SRCREV_cert-renewal = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/cert/cert-renewal"
+SRCREV_FORMAT .= "_config-common"
+SRCREV_config-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/config-common"
+SRCREV_FORMAT .= "_http-common"
+SRCREV_http-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/http-common"
+SRCREV_FORMAT .= "_logger"
+SRCREV_logger = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/logger"
+SRCREV_FORMAT .= "_openssl-build"
+SRCREV_openssl-build = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/openssl-build"
+SRCREV_FORMAT .= "_openssl-sys2"
+SRCREV_openssl-sys2 = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/openssl-sys2"
+SRCREV_FORMAT .= "_openssl2"
+SRCREV_openssl2 = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/openssl2"
+SRCREV_FORMAT .= "_pkcs11"
+SRCREV_pkcs11 = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/pkcs11/pkcs11"
+SRCREV_FORMAT .= "_pkcs11-sys"
+SRCREV_pkcs11-sys = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/pkcs11/pkcs11-sys"
+SRCREV_FORMAT .= "_test-common"
+SRCREV_test-common = "b9fff6b2fdf9c1593a5a0b7e856a5f01c2c5ad5b"
+EXTRA_OECARGO_PATHS += "${WORKDIR}/test-common"
```

Comment:
Same as previous comment...we used to delete these autogenerated SRCREV_FORMAT, SRCREV_*, and EXTRA_OECARGO_PATHS variable assignments from the .bb file. Do we not need to do that anymore? The less changes we need to make the better IMO, but I wanted to double check. :)